annotations.md

```diff
@@ -28,7 +28,16 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.revision** Source control revision identifier for the packaged software.
 * **org.opencontainers.image.vendor** Name of the distributing entity, organization or individual.
 * **org.opencontainers.image.licenses** License(s) under which contained software is distributed as a [SPDX License Expression][spdx-license-expression].
-* **org.opencontainers.image.ref.name** Name of the reference for a target (string). SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+* **org.opencontainers.image.ref.name** Name of the reference for a target (string).
+  * SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+  * Character set of the value SHOULD conform to alphanum of `A-Za-z0-9` and separator set of `-._:@/+`
+  * An EBNF + regular expression like:
+    ```
+    ref := component ["/" component]*
```

Comment:
Both [ABNF][1] and [EBNF][2] (§4.3 and table 1) use `=` for rules, so I'd recommend `:=` → `=`.

Both [ABNF][3] and [EBNF][2] (§4.12) use a *prefixed* `*` for repetition (not a suffixed `*`), so I'd recommend `["/" component]*` → `*["/" component]`.

[1]: https://tools.ietf.org/html/rfc5234#section-2.2
[2]: http://standards.iso.org/ittf/PubliclyAvailableStandards/s026153_ISO_IEC_14977_1996%28E%29.zip
[3]: https://tools.ietf.org/html/rfc5234#section-3.6